test/integration/steps_test.js

```diff
@@ -468,7 +575,7 @@ describe('Integration | steps.js', () => {
       });
     });
 
-    it('should not fail when database contains plpgsql source', async function() {
+    it('should not fail when database contains pgsql source', async function() {
```

Comment:
ahah ! c'est bien `plpgsql` le language procédural de PostgreSQL dont il est question ici
```suggestion
    it('should not fail when database contains plpgsql source', async function() {
```